data/mil2525d/core_data/gdbs/README.md

```diff
@@ -43,14 +43,19 @@ To view the Label Rules and associations, view the symbol dictionary table "Labe
 | ------------- | -------------- | --- | ----------------- | ----- |
 | additionalinformation  | H | H | Additional Information | |
 | additionalinformation2 | H | H | Additional Information 2 | Used in the Convoys and Airspace Coordination Area tactical graphics. |
+| azimuth | AN | AN | Azimuth | |
```

Comment:
After giving this table a closer look (after not looking at for a long while) and while we are updating this table, we should probably fix all of the inconsistencies: (1) sort table by the Field ID (2) remove the "2525C" column (3) make the ones like "T1" "Unique Designation 2" consistent (4) fix anything that looks weird or makes the table hard to use